snippets/native-platforms/ios-objc/dependencies.md

```diff
@@ -1,4 +1,4 @@
 ```ruby
-pod 'Lock', '~> 1.12'
-pod 'JWTDecode', '~> 0.2'
+pod 'Lock', '~> 1.16'
+pod 'JWTDecode', '0.2'
```

Comment:
I believe this is because `0.3.0` drops support for iOS 7. In this case let's specify the full number of the version which is `0.2.2` or even use `~> 0.2.2` if we release a new patch for `0.2`